### DISCUSSION TITLE
Update documentation on user authentication & creation

### DISCUSSION THREAD

**Discussion by rizkysyazuli** [firebase/emberfire#530] [Ideas]
I just spent some time scratching my head wondering why i can't create new user account.

I'm using emberfire (and firebase) for the first time. After seeing the docs on authentication, i was under the assumption that using the password provider would also create a new user account when none user exists in the database. I was wrong..

I only found out later on after reading the migration guide and seeing this piece of code.

```
firebaseApp: Ember.inject.service(),

actions: {
  createUser(email, pass) {
    const auth = this.get('firebaseApp').auth();
    auth.createUserWithEmailAndPassword(email, pass).then((userResponse) => {
      const user = this.store.createRecord('user', {
        id: userResponse.uid,
        email: userResponse.email
      });
      return user.save();
    });
  }
}
```

Seriously, the migration guide is likely the last thing a **new user** would look. Especially when you already put a compatibility table over there.

Plus, the response data doesn't contain a `uid`. What i got was `localId`.

**Comment by davidchalifoux** [comment]
Why not make a pull request to fix this?
